homeassistant/components/zhong_hong/climate.py

```diff
@@ -46,7 +47,21 @@
     }
 )
 
-SUPPORT_HVAC = [HVAC_MODE_COOL, HVAC_MODE_HEAT, HVAC_MODE_DRY, HVAC_MODE_FAN_ONLY]
+SUPPORT_HVAC = [
+    HVAC_MODE_COOL,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_DRY,
+    HVAC_MODE_FAN_ONLY,
+    HVAC_MODE_OFF,
+]
+
+
+MODE_TO_STATE = {
+    HVAC_MODE_COOL: HVAC_MODE_COOL,
```

Comment:
If we use the same constants on both sides this dict doesn't give the decoupling that is its purpose. Please define new constants for the device modes and use here.